includes/Verification/VerificationLookup.php

```diff
@@ -75,7 +75,25 @@ public function verificationEntityFromTitle( Title $title ): ?VerificationEntity
 	public function verificationEntityFromQuery( array $query ): ?VerificationEntity {
 		$res = $this->lb->getConnection( DB_REPLICA )->selectRow(
 			static::TABLE,
-			[ '*' ],
+			[
```

Comment:
I think we need to add a comment in `sql/data_accounting.sql` to remind anyone who add a column to also add it here.

Reply:
And also say it for WitnessLookup.php.